custom_components/wyzeapi/__init__.py

```diff
@@ -147,10 +134,11 @@ def get_uid():
         device_registry, config_entry.entry_id
     ):
         for identifier in device.identifiers:
+            # domain has to remain here. If it is removed the integration will remove all entities for not being in the mac address list each boot.
             domain, mac = identifier
             if mac not in mac_addresses:
                 _LOGGER.warning(
-                    f"{mac} is not in the mac_addresses list. Removing the entry..."
+                    '%s is not in the mac_addresses list, removing the entry', mac
```

Comment:
Is this just a style choice? Not a problem either way

Reply:
The home assistant lint configuration says that is a slight optimization. Basically the lazy formatting only happens if it needs to log. Normal substitutions happen always.